agora/Agora/Treasury.hs

```diff
@@ -40,13 +38,11 @@ data TreasuryRedeemer
       Show
     , -- | @since 0.1.0
       GHC.Generic
+    , Enum
+    , Bounded
```

Comment:
Needs `@since` tagging now... Specifically targeting `0.2.0`. Arguably, there will be some backwards incompatible things, but since it's pre-1.0, we don't care about API stability (and don't promise it)